src/main/java/driver/DriverFactory.java

```diff
@@ -67,17 +67,19 @@ public void filterByStatusInactive() {
         wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("device-filter-apply-button")));
         driver.findElement(By.id("device-filter-apply-button")).click();
     }
-
     public String getDeviceName() {
-        String firstRowText = driver.findElement(By.id("device-table-row-0")).getText();
+        String firstRowText = driver.findElement(By.id("device-table")).getText();
         String[] attributes = firstRowText.split("\n");
         return attributes[0];
     }
```

Comment:
is this working as intended? becouse you take the text from the whole table and then return the first element which if im not mistaken should be a text "Name"